app/graphql/types/objects/consultation/base.rb

```diff
@@ -99,6 +100,14 @@ def responses(response_token:, sort:, sort_direction:)
 					return object.responses.acceptable_responses.sort_records(sort, sort_direction)
 				end
 
+				def consultation_partner_responses
```

Comment:
```suggestion
				def consultation_partner_responses
```

From what I understand, this does not return the responses, but only the counts, right?

Reply:
@mikevic Yes, {id, count} grouping

Reply:
What I suggest is to rename this to `consultation_partner_response_counts`, so that it is clear

Reply:
![image](https://github.com/user-attachments/assets/d0799e01-6a49-4861-a0fd-c122a6015f24)
It also outputs Organisation Object too